### DISCUSSION TITLE
Monthly Meeting: 4. July 2023, 2pm CET

### DISCUSSION THREAD

**Discussion by KaiVolland** [geostyler/geostyler#2287] [Show and tell]
# Topics

- TODOS v13.1.0:
  - Composable props:
    - Missing `FilterTree` props
    - Missing `PreviewMap` props
    - Remove `showAmount` and `showDuplicates` from `Rules`
  - Transform FilterTree Ui to look like FunctionUI

**Comment by dnlkoch** [comment]
Monthly has been cancelled.
